sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessage.java

```diff
@@ -94,7 +94,6 @@ public ServiceBusMessage(ServiceBusReceivedMessage receivedMessage) {
 
         this.amqpAnnotatedMessage = new AmqpAnnotatedMessage(receivedMessage.getAmqpAnnotatedMessage());
```

Comment:
I am wary of this copy constructor. It is not doing a deep copy and also not sure if we need this constructor for `AmqpAnnotatedMessage`.